ui/src/schemas/index.ts

```diff
@@ -147,6 +148,18 @@ export const telegraf = new schema.Entity(
 
 export const arrayOfTelegrafs = [telegraf]
 
+/* Templates */
+
+// Defines the schema for the "templates" resource
+export const template = new schema.Entity(
+  ResourceType.Templates,
+  {},
+  {
+    processStrategy: (t: TemplateSummary): TemplateSummary => t,
```

Comment:
Don't need a `processStrategy` if we aren't modifying the object in some way.

Reply:
I'm surprised to see that we are not modifying the object, especially since the BE guarantees very little about templates?

Reply:
I put that in there as a placeholder. It will be updated.